internal/microsoft_edge.py

```diff
@@ -68,8 +70,18 @@ def prepare(self, job, task):
         self.bodies_path = os.path.join(task['dir'], 'bodies')
         if not os.path.isdir(self.bodies_path):
             os.makedirs(self.bodies_path)
-        if not task['cached']:
-            self.clear_cache()
+        try:
+            import _winreg
+            registry_key = _winreg.CreateKeyEx(_winreg.HKEY_CURRENT_USER, self.edge_registry_path, 0, _winreg.KEY_READ | _winreg.KEY_WRITE)
+            self.edge_registry_key_value = _winreg.QueryValueEx(registry_key, "ClearBrowsingHistoryOnExit")[0]
+            if task['cached'] or job['fvonly']:
+                _winreg.SetValueEx(registry_key, "ClearBrowsingHistoryOnExit", 0, _winreg.REG_SZ, "1")
```

Comment:
Everything I can find online says it is a DWORD value key (which is normal for flags).  Unless you can find documentation that shows it is a string key, all of these should be writing it as a DWORD.

Reply:
Will change and test it on monday, when i have access to the windows10 system.